mysql/metadata.csv

```diff
@@ -42,3 +42,108 @@ mysql.replication.seconds_behind_master,gauge,,second,,The lag in seconds betwee
+mysql.innodb.data_fsyncs,gauge,,operation,second,"The number of fsync() operations per second.",0,mysql,mysql innodb data_fsyncs
+mysql.innodb.data_pending_fsyncs,gauge,,operation,,"The current number of pending fsync() operations.",0,mysql,mysql innodb data_pending_fsyncs
+mysql.innodb.data_pending_reads,gauge,,read,,"The current number of pending reads.",0,mysql,mysql innodb data_pending_reads
+mysql.innodb.data_pending_writes,gauge,,write,,"The current number of pending writes.",0,mysql,mysql innodb data_pending_writes
+mysql.innodb.data_read,gauge,,byte,second,"The amount of data read per second.",0,mysql,mysql innodb data_read
+mysql.innodb.data_written,gauge,,byte,second,The amount of data written per second.,0,mysql,mysql innodb data_written
+mysql.innodb.dblwr_pages_written,gauge,,page,second,"The number of pages written per second to the doublewrite buffer.",0,mysql,mysql innodb dblwr_pages_written
+mysql.innodb.dblwr_writes,gauge,,byte,second,"The number of doublewrite operations performed per second.",0,mysql,mysql innodb dblwr_writes
+mysql.innodb.history_list_length,gauge,,,,"History list length as shown in the TRANSACTIONS section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb history_list_length
+mysql.innodb.log_waits,gauge,,wait,second,"The number of times that the log buffer was too small and a wait was required for it to be flushed before continuing.",0,mysql,mysql innodb log_waits
+mysql.innodb.log_write_requests,gauge,,write,second,"The number of write requests for the InnoDB redo log.",0,mysql,mysql innodb log_write_requests
+mysql.innodb.log_writes,gauge,,write,second,"The number of physical writes to the InnoDB redo log file.",0,mysql,mysql innodb log_writes
+mysql.innodb.lsn_current,gauge,,,,"Log sequence number as shown in the LOG section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb lsn_current
+mysql.innodb.lsn_flushed,gauge,,,,"Flushed up to log sequence number as shown in the LOG section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb lsn_flushed
+mysql.innodb.lsn_last_checkpoint,gauge,,,,"Log sequence number last checkpoint as shown in the LOG section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb lsn_last_checkpoint
+mysql.innodb.mem_adaptive_hash,gauge,,,,"As shown in the BUFFER POOL AND MEMORY section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb mem_adaptive_hash
+mysql.innodb.mem_dictionary,gauge,,,,"As shown in the BUFFER POOL AND MEMORY section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb mem_dictionary
+mysql.innodb.mem_total,gauge,,,,"As shown in the BUFFER POOL AND MEMORY section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb mem_total
+mysql.innodb.os_log_pending_fsyncs,gauge,,operation,,"Number of pending InnoDB log fsync (sync-to-disk) requests.",0,mysql,mysql innodb os_log_pending_fsyncs
```

Comment:
```suggestion
mysql.innodb.os_log_pending_fsyncs,gauge,,request,,"Number of pending InnoDB log fsync (sync-to-disk) requests.",0,mysql,mysql innodb os_log_pending_fsyncs
``` 
?

Reply:
I think that `operation` better fits here, the request part is just because the operation is pending.